mpi_ring_c_test.sh

```diff
@@ -24,6 +25,9 @@ else
     exit 1
 fi
 
+# Fortran compile test
```

Comment:
I think we need to test both compilation and mpirun.

Reply:
Adding